components/typography.css

```diff
@@ -0,0 +1,42 @@
+h1 {
+	font-weight: bold;
+	font-size: 40px;
+	line-height: 55px;
+}
+
+h2 {
+	font-weight: 800;
+	font-size: 32px;
+	line-height: 44px;
+}
+
+h3 {
+	font-weight: bold;
+	font-size: 28px;
+	line-height: 38px;
+}
+
+h4 {
+	font-weight: 600;
+	font-size: 28px;
+	line-height: 38px;
```

Comment:
you can put these things in rem instead of px

Reply:
A must to do I will do it asap,thanks for pointing